src/main/java/br/com/sbs/cubatech/category/CategoryApiController.java

```diff
@@ -0,0 +1,25 @@
+package br.com.sbs.cubatech.category;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class CategoryApiController {
+
+    private CategoryRepository categoryRepository;
```

Comment:
Esse atributo pode ser `final`.

```suggestion
    private final CategoryRepository categoryRepository;
```